### DISCUSSION TITLE
Hide/show +Rule button based on Field select

### DISCUSSION THREAD

**Discussion by kellywang1234** [react-querybuilder/react-querybuilder#670] [General]
Hi, is it possible to hide/show +Rule button based on Field change? https://codesandbox.io/p/sandbox/react-query-builder-two-value-valueeditor-forked-6dk894?file=%2Fsrc%2FApp.tsx%3A85%2C42 In this example, when we select "Hide rule button" field option, the +Rule button should disabled/hide. I tired addRuleAction but it seems the behaviors happens when on click rule rather than field change. Thanks in advance!

**Comment by jakeboone02** [comment]
The "+ Rule" button is at the group level, so I'm not sure how the logic would be based on a rule-level property like `field`. Also I can't access the sandbox; did you move it out of drafts?

**Comment by kellywang1234** [comment]
sorry about the sandbox, but i figured out the issue! thank you anyway, i will close this discussion.
